### PR TITLE
sql analytical store add bicep support

### PR DESCRIPTION
**Pull request by markjbrown**
# PR Checklist

## Changelog

* add bicep support

### CODE REVIEW

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+@minValue(4000)
+@maxValue(1000000)
+param autoscaleMaxThroughput int = 4000
+
+@description('Time to Live for data in analytical store. (-1 no expiry)')
+@minValue(-1)
+@maxValue(2147483647)
+param analyticalStoreTTL int = -1
+
+var accountName_var = toLower(accountName)
```
[the remark this answers is not in the document]
Comment (markjbrown) [reply quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep:38]: @bmoore-msft is there a naming convention I should follow for these when I have both a parameter and a variable with the same name? Not sure what the right way to distinguish these other then calling them _var, _param, _resource.

[the remark this answers is not in the document]
Comment (markjbrown) [reply quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep]: @bmoore-msft ok. Cleaned up the variable names in here. Should be good to go I think.

File: quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    Throughput: manualProvisionedThroughput
+  }
+  Autoscale: {
+    autoscaleSettings: {
+      maxThroughput: autoscaleMaxThroughput
+    }
+  }
+}
+
+resource accountName_resource 'Microsoft.DocumentDB/databaseAccounts@2021-04-15' = {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep:56]: remove the _resource from the symbolic names...  I think in this sample you can just use the resource type, eg.

databaseAccount
sqlDatabase
container

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
Sorry for the delay - was on vacation... a few more decompile things...

File: quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep (hunk 2 above)
Comment (markjbrown) [reply quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep]: fixed in new commit.

**Comment by markjbrown** [comment]
@bmoore-msft this is ready to go.

**Pull request merged**